Assets/Global/Scripts/RoomLogic/RoomTransitionDoor.cs

```diff
@@ -16,18 +16,27 @@ public class RoomTransitionDoor : Interactable
     [SerializeField] private Animator animator;
     [SerializeField] private MeshRenderer doorMesh;
     [SerializeField] private string nextRoomName;
-    private bool isTransitioning = false;
+    public int nextRoomid;
+
     public CrossfadeController crossfadeController;
     public PlayerSpawnPoint playerSpawnPoint;
+    public DoorManager doorManager;
```

Comment:
check for line 19 till 23 if public is really what you want and also look if you want to add [HideInInspector] for people who dont need to see it in the inspector